frontend/src/components/ProfileDropdown/ProfileDropdown.tsx

```diff
@@ -0,0 +1,27 @@
+import classes from './ProfileDropdown.module.scss'
+import { Award, Handshake, LogOut, Map } from 'lucide-react'
+import ProfileDropDownItems from '../ProfileDropdownItems/ProfileDropdownItems'
+
+
+function ProfileDropdown (props: {username: string}) {
+  return (
+    <div className={classes.dropDownMenu}>
+      <h3 className={classes.username}>{props.username}</h3>
+      
+      <ProfileDropDownItems href='/' text='Levels'>
+        <Map color='hsl(52, 60%, 50%)' strokeWidth={3}/>
+      </ProfileDropDownItems>
+      <ProfileDropDownItems href='/' text='Friends'>
```

Comment:
probably get rid of this I don't think we plan on implementing this functionality and if we do we can add this back

Reply:
do you mean remove the 'Levels' option from the dropdown menu?

Reply:
I meant friends actually but to be fair idk what the levels button would be either. Maybe if you could replace those two with a profile button?  (as in the button that presents user information etc.)